test/support/example_behaviour.ex

```diff
@@ -221,14 +221,14 @@ end
 
 defmodule ElixirSenseExample.ExampleBehaviourWithDocCallback do
   @behaviour ElixirSenseExample.ExampleBehaviourWithDoc
-  @behaviour GenServer
```

Comment:
Those behaviours are a test actually https://github.com/elixir-lsp/elixir_sense/pull/84/commits/9100679845ff111bde15bef3c00019d59f37d55c
IMO it's better to refactor this

Reply:
Would you please be so kind as to help me with that? Feel free to add commits to the branch.

Reply:
Done on master

Reply:
Thanks, @lukaszsamson